src/controllers/taskUpdatesHandler.ts

```diff
@@ -0,0 +1,25 @@
+import { env } from "../typeDefinitions/default.types";
+import JSONResponse from "../utils/JsonResponse";
+import * as response from "../constants/responses";
+import { verifyNodejsBackendAuthToken } from "../utils/verifyAuthToken";
+import { sendTaskUpdate } from "../utils/sendTaskUpdates";
+
+export const sendTaskUpdatesHandler = async (request: any, env: env) => {
+  try {
+    const authHeader = request.headers.get("Authorization");
+    if (!authHeader) {
+      return new JSONResponse(response.BAD_SIGNATURE);
+    }
+    await verifyNodejsBackendAuthToken(authHeader, env);
+    const updates: {
+      content: { completed: string; planned: string; blockers: string };
+    } = await request.json();
+    const { completed, planned, blockers } = updates.content;
+    await sendTaskUpdate(completed, planned, blockers, env);
+    return new JSONResponse(
+      "Task update sent on discord tracking updates channel."
+    );
```

Comment:
Is the try-catch correctly performed here? Seems like any exception will end up returning bad signature reason, and that may be misleading.

Reply:
Yes, this should be fixed to an internal server error.
Please check now.